drivers/shared/executor/executor_universal_linux.go

```diff
@@ -173,19 +174,21 @@ func DestroyCgroup(groups *lconfigs.Cgroup, executorPid int) error {
 	return mErrs.ErrorOrNil()
 }
 
-func (e *UniversalExecutor) start(command *ExecCommand) error {
-	if command.NetworkIsolation != nil && command.NetworkIsolation.Path != "" {
+// wrapNetns takes a given function and calls it inside the network namespace
```

Comment:
nitpicking language: probably more concise to say: 
```suggestion
// wrapNetns calls passed function, inside the network namespace `spec`.
```